templates/telemetry/heartbeat/heartbeat.4.schema.json

```diff
@@ -0,0 +1,82 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "heartbeat",
+  "type": "object",
+  "properties": {
+    @TELEMETRY_APPLICATION_1_JSON@,
+    @TELEMETRY_CLIENTID_1_JSON@,
+    @TELEMETRY_CREATIONDATE_1_JSON@,
+    @TELEMETRY_ID_1_JSON@,
+    "payload": {
+      "additionalProperties": false,
+      "properties": {
+        "version": {
+          "maximum": 1,
+          "minimum": 1,
+          "type": "number"
+        },
+        "flowId": {
+          "type": "string"
+        },
+        "offeredTS": {
```

Comment:
Please add a tighter bound here, at least `>0` (and for the rest of the TS fields).

Reply:
Added bounds of >2010 and <2100